addon/mixins/application-route-mixin.js

```diff
@@ -11,20 +11,17 @@ export default Ember.Mixin.create(ApplicationRouteMixin, {
     var state = documentSource.get('state'); // Forces creation of the state, which clears the query params
     var isStatePresent = state.get('isOpen') || state.get('isCreate');
 
-    if (!isStatePresent) {
-      return;
+    if (isStatePresent) {
+      this.get('session').authenticate('authenticator:gdrive', { 'login_hint': state.get('userID') }).then(function () {
```

Comment:
Oh here also. Never mind I see it. I think previous versions of ember-simple-auth didn't support the 2nd param.